examples/face_recognition/README.md

```diff
@@ -0,0 +1,5 @@
+# WebNN API Face Recognition Example
+This example loads Face Detection models (trained by WIDERFace) and Face Recognition model (trained by davidsandberg) in TensorFlow format, constructs and inferences them by WebNN API.
```

Comment:
Are both Facenet and Face Reidentification Retail models trained by davidsandberg? Are they in TensorFlow format?

Reply:
oops! I forgot to update this file, sorry.